docs/changelog/1_0_rc3.rst

```diff
@@ -0,0 +1,149 @@
+========
+1.0 RC 3
+========
+
+This changelog summarizes changes and bugfixes in EdgeDB 1.0 Release
+Candidate 3 "Cygni". This release is focusing on fixing existing
+issues rather than introducing new features.
+
+
+Packaging
+=========
+
+The biggest change in this release is in how EdgeDB is packaged. We
+now have a "portable build" that runs on any non-legacy Linux or macOS
+system. The purpose is to avoid restrictions that supporting native
+packaging systems imposes. Because of this we recommend a more
+explicit ``dump``/``restore`` upgrade workflow to get your instances
+to RC3.
+
+Before you start you may want to backup the instance credentials
+(which you can find in the "Config" directory indicated by
+:ref:`ref_cli_edgedb_info`). The first step in the upgrade process
+will be backing up all the instance data and that will include the
+login credentials that the instance expects. After restoring the
+database dump, you may need to restore old credentials file as
+well.
+
+Project upgrade
+---------------
+
+If you have an EdgeDB project, you can upgrade it to RC3 using the
+following steps from the project directory:
+
+.. code-block:: bash
+
+    $ edgedb dump --all --format=dir upgrade_backup
+    $ edgedb instance destroy --force my_instance
+    $ edgedb project upgrade --to-latest
+    $ edgedb project init --no-migrations
+    $ edgedb restore --admin --all upgrade_backup
+
+Now that the project has been re-initialized and data dump is
+restored, we need to restore the login credentials. Which presents a
+couple of options. We can restore the old credentials:
+
+.. code-block:: bash
+
+    $ edgedb instance reset-password my_instance --password-from-stdin
+
+Alternatively we can create new credentials. This is recommended if
```

Comment:
Let's put the recommended option first and suggest `--password-from-stdin` iff `reset-password` was called on an instance previously and the password needs to be kept.